frontend/packages/console-dynamic-plugin-sdk/scripts/package-definitions.ts

```diff
@@ -158,7 +158,7 @@ export const getWebpackPackage: GetPackageDefinition = (
       ),
       ...parseDeps(
         rootPackage,
-        ['ajv', 'chalk', 'comment-json', 'find-up', 'read-pkg', 'semver'],
+        ['ajv', 'chalk', 'comment-json', 'find-up', 'glob', 'read-pkg', 'semver'],
```

Comment:
This essentially adds `glob` to `@openshift-console/dynamic-plugin-sdk-webpack` package dependencies.